CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 1.0.0-beta1
+
+This is the first beta release of this client.
```

Comment:
it worth mentioning that the client is now supports kubernetes 1.8 clusters. otherwise looks good.

Reply:
Oh good call